org.blackcoinmore.blackmore-qt.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id": "org.blackcoinmore.blackmore-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "command": "blackmore-qt",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "blackcoin-more",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0744 -D -t /app/bin/internal bin/blackmored",
+                "install -m 0744 -D -t /app/bin/internal bin/blackmore-qt",
+                "install -m 0744 -D -t /app/bin/internal bin/blackmore-cli",
+                "install -m 0744 -D -t /app/bin/internal bin/blackmore-tx"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/CoinBlack/blackcoin-more/releases/download/v13.2.1/blackcoin-more-13.2.1-x86_64-linux-gnu.tar.gz",
+                    "sha256": "3681da3217019e202b887ab5d34f2a5ad3a1dd98368d3c7362f72753c0722184"
+                },
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "aarch64"
+                    ],
+                    "url": "https://github.com/CoinBlack/blackcoin-more/releases/download/v13.2.1/blackcoin-more-13.2.1-aarch64-linux-gnu.tar.gz",
+                    "sha256": "bd2a5f0cacfe170cdae814f64e41f1904a08911da4b09629b2441d74166dc780"
+                }
+            ]
+        },
+        {
+            "name": "misc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "desktop-file-edit --set-key=\"Icon\" --set-value=${FLATPAK_ID} blackmore-qt.desktop",
+                "install -Dp -m 644 blackmore-qt.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dp -m 644 blackcoin.svg /app/share/icons/hicolor/scalable/apps/org.blackcoinmore.blackmore-qt.svg",
+                "install -Dp -m 644 org.blackcoinmore.blackmore-qt.metainfo.xml /app/share/metainfo/org.blackcoinmore.blackmore-qt.metainfo.xml",
+                "bash install_wrappers.sh"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/coinblack/blackmore-flatpak/main/blackmore-qt.desktop",
```

Comment:
you should pin this to the tag.

Reply:
same as mentioned before, the files will be added to the Blackcoin More repo